ui/components/ui/list-item/index.scss

```diff
@@ -110,3 +110,18 @@
       '.    actions  actions  actions  actions  mid mid mid mid right right right';
   }
 }
+
+.no-subtitle {
+  grid-template-areas:
+    'icon head     head     head     head     head     head     head     right right right right'
+    'icon head     head     head     head     head     head     head     right right right right'
+    'icon head     head     head     head     head     head     head     right right right right';
```

Comment:
this solution is great, but we can't use the lack of secondary prop to nix the 'actions' row (this is reserved for speed up / cancel buttons on transaction/activity items that are pending).

So instead of no-subtitle, lets try something like 'single-content-row' and reduce the grid template areas to a single row. So if !children && !secondary instead of just !secondary

Reply:
Yes this class is only being added to asset-list-item so I don't think it could cause the issue as implemented (no actions but I see your point and will make this change in case someone tries to use it in a different context!